src/main/java/edu/tamu/cap/utility/TripleUtil.java

```diff
@@ -0,0 +1,68 @@
+package edu.tamu.cap.utility;
+
+import java.io.ByteArrayInputStream;
+import java.io.InputStream;
+import java.nio.charset.StandardCharsets;
+
+import org.apache.jena.rdf.model.Model;
+import org.apache.jena.rdf.model.ModelFactory;
+import org.apache.jena.riot.RDFDataMgr;
+import org.apache.jena.riot.RDFLanguages;
+
+import edu.tamu.cap.exceptions.InvalidTripleException;
+import edu.tamu.cap.model.response.Triple;
+
+public class TripleUtil {
+
+    /**
+     * Convert and validate an Cap Triple to a Apache Jena Triple.
+     *
+     * This uses the triple language N3 because it fully complies with the RDF specification in regards to a triple and UTF-8.
+     * When validation fails, InvalidTripleException is thrown.
+     *
+     * @param triple
+     *   The triple to validate that it has the proper format and is properly escaped.
+     *
+     * @return
+     *   The generated triple.
+     *
+     * @throws InvalidTripleException
+     */
+    public static org.apache.jena.graph.Triple toJenaTriple(Triple triple) throws InvalidTripleException {
+        Model model =  ModelFactory.createDefaultModel();
```

Comment:
https://jena.apache.org/documentation/javadoc/jena/org/apache/jena/graph/NodeFactory.html -> static method `createURI` and `createLiteral`

https://jena.apache.org/documentation/javadoc/jena/org/apache/jena/graph/Triple.html -> static method `create`

Something like below should work:

```
org.apache.jena.graph.Triple jenaTriple = create(createURI(triple.getSubject()), createURI(triple.getPredicate()), createLiteral(triple.getObject));
```

Reply:
I've tried those originally and the result in double-escaping.

Those methods actually treat the input as unescaped and will escape the String.
I was also unable to find a way to tell them to not do that.

We simply cannot use those or we have to change the design to never send/receive escaped data to/from the front-end.